src/OpenTelemetry.Api/AssemblyInfo.cs

```diff
@@ -11,6 +11,8 @@
 [assembly: InternalsVisibleTo("OpenTelemetry.Exporter.InMemory" + AssemblyInfo.PublicKey)]
 [assembly: InternalsVisibleTo("OpenTelemetry.Exporter.OpenTelemetryProtocol" + AssemblyInfo.PublicKey)]
 [assembly: InternalsVisibleTo("OpenTelemetry.Exporter.OpenTelemetryProtocol.Tests" + AssemblyInfo.PublicKey)]
+[assembly: InternalsVisibleTo("OpenTelemetry.Exporter.Zipkin" + AssemblyInfo.PublicKey)]
+[assembly: InternalsVisibleTo("OpenTelemetry.Exporter.Zipkin.Tests" + AssemblyInfo.PublicKey)]
```

Comment:
This change needs to be discussed by maintainers.
We've historically been hesitant to make internals visible to other libraries.
I understand there's precedence for other Exporters (OTLP and Console and InMemory), but I don't know the longterm plans for the Zipkin Exporter.

Reply:
This was discussed in the 2/4 SIG meeting.

The decision is that InternalsVisibleTo is the wrong direction to go. The fact that internals are visible to other libraries is considered technical debt and the goal is to eventually remove these.